### PR TITLE
Mets transformer gets a vhs key from the adapter and reads xml from VHS

### PR DESCRIPTION
**Pull request by alicefuzier**

### CODE REVIEW

File: pipeline/transformer/transformer_mets/src/test/scala/uk/ac/wellcome/platform/transformer/mets/parsers/MetsXmlParserTest.scala [hunk 1]
```diff
@@ -29,75 +29,64 @@ class MetsXmlParserTest extends FunSpec with Matchers {
...
+  def xml =
+    IOUtils.toString(getClass.getResourceAsStream("/b30246039.xml"), "UTF-8")
 
   def xmlNodmdSec =
-    s"""
-       |<mets:mets xmlns:mets="http://www.loc.gov/METS/" xmlns:mods="http://www.loc.gov/mods/v3">
-       |  <mods:recordIdentifier source="gbv-ppn">b30246039</mods:recordIdentifier>
-       |</mets:mets>
-       |""".stripMargin.toInputStream
+    <mets:mets xmlns:mets="http://www.loc.gov/METS/" xmlns:mods="http://www.loc.gov/mods/v3">
```
Comment (jamesgorrie) [inline pipeline/transformer/transformer_mets/src/test/scala/uk/ac/wellcome/platform/transformer/mets/parsers/MetsXmlParserTest.scala]: https://contributors.scala-lang.org/t/proposal-to-remove-xml-literals-from-the-language/2146/94
🤷‍♂

Comment (alicefuzier) [reply pipeline/transformer/transformer_mets/src/test/scala/uk/ac/wellcome/platform/transformer/mets/parsers/MetsXmlParserTest.scala]: I know, but I think it's an easy fix when we'll have to upgrade scala. In the meantime, it makes code a bit more maintainable and readable

File: pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/service/MetsTransformerWorkerService.scala [hunk 2]
```diff
@@ -33,25 +30,28 @@ class MetsTransformerWorkerService(
...
+      _ <- messageSender.sendT(work).toEither
     } yield ()
   }
 
-  private def fetchMets(metsData: MetsData) = {
-    Future {
-      s3Client.getObject(s3Config.bucketName, metsData.path).getObjectContent
+  private def fetchMets(key: Version[String, Int]) = {
+    store.get(key) match {
+      case Left(err)                   => Left(new Exception(err.e))
```
Comment (warrd) [inline pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/service/MetsTransformerWorkerService.scala]: Is there any need to wrap within a new exception? I think `err.e` is an exception itself?

Comment (alicefuzier) [reply pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/service/MetsTransformerWorkerService.scala]: You're right, no need

**Pull request merged**